spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::UsersController, type: :controller do
+  let!(:user) { create :user }
+  let(:user_id) { user.id }
+  let(:user_email) { user.email }
+
+  describe 'GET #show' do
+    before { get :show, params: { id: user_id } }
+
+    context 'when there the user exists' do
+      it 'get an user' do
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'returns the user' do
+        expect(json).not_to be_empty
+        expect(json['id']).to eq(user_id)
+        expect(json['email']).to eq(user_email)
+      end
+    end
```

Comment:
Here you are not testing that the endpoint returns an user, you're testing that the json have an id and an email. Maybe you can test here that the response contains the user serialized with the serializer, what do you think?